python/bindings/openravepy_kinbody.cpp

```diff
@@ -1001,7 +1001,7 @@ JointControlInfo_IOPtr PyJointControlInfo_IO::GetJointControlInfo()
     size_t num1, num2;
     if( !IS_PYTHONOBJECT_NONE(moveIONames) ) {
         num1 = len(moveIONames);
-        OPENRAVE_EXCEPTION_FORMAT0(num1 == info.moveIONames.size(), ORE_InvalidState);
+        OPENRAVE_ASSERT_FORMAT0(num1 == info.moveIONames.size(), _("unexpected size"), ORE_InvalidState);
```

Comment:
@lazydroid i think these should also be `<=` check.

Reply:
changed, thank you!

Reply:
@yoshikikanemoto still looks good!

![image](https://github.com/user-attachments/assets/bf97ed59-919a-40c7-ad57-3a4680d91241)